src/pages/markets/components/sections/_nav-tab.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import { Text } from 'components/elements'
+import { Flex } from 'components/containers'
+import { Localize, LocalizedLink } from 'components/localization'
+// import { useActiveLinkState } from 'components/hooks/use-active-link-state'
```

Comment:
👀

Reply:
removed the line in the current commit..